File: tests/fixtures/languages/sample.dart
// Minimal Dart fixture for language parse-validation tests.

import 'dart:async';

class Greeter {
  final String label;

  Greeter(this.label);

  String greet() => 'Hello, $label';
}

mixin Doubler {
  int doubled();
}

enum Status { active, idle, done }

int add(int a, int b) {
  return a + b;
}

File: tests/fixtures/languages/sample.php
<?php
// Minimal PHP fixture for language parse-validation tests.

namespace Demo;

use SplStack;

class Greeter {
    public function __construct(private string $label) {}

    public function greet(): string {
        return "Hello, {$this->label}";
    }
}

interface Doubler {
    public function doubled(): int;
}

function add(int $a, int $b): int {
    return $a + $b;
}
